### PR TITLE
feat: biw project edit correctly

### PR DESCRIPTION
**Pull request by Suduck**
## What does this PR change?

This PR introduces the ability to edit project in a better way, there is still a long way to go but we are getting closer

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/biw-project-edit
2. Open the builder in world panel and create a new project

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/ManifestTranslator.cs [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+            AssetCatalogBridge.i.AddScenesObjectToSceneCatalog(assets);
+            
+            //We create and assign the data of the scene
+            LoadParcelScenesMessage.UnityParcelScene parcelData = new LoadParcelScenesMessage.UnityParcelScene();
+            parcelData.id = manifest.scene.id;
+            
+            //We set the current scene in the 0,0
+            int x = 0;
+            int y = 0;
+            // int y = CommonScriptableObjects.playerCoords.Get().y;
```
Comment (sandrade-dcl) [inline unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/ManifestTranslator.cs:238]: Lets remove the commented code!

File: unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/ManifestTranslator.cs [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+                        
+                        case "GLTFShape":
+                            LoadableShape.Model gltfModel = JsonConvert.DeserializeObject<LoadableShape.Model>(component.data.ToString());
+                            EntityComponentsUtils.AddGLTFComponent(scene, entity, gltfModel, component.id);
+                            break;
+                        
+                        case "NFTShape":
+                            //Builder use a different way to load the NFT so we convert it to our system
+                            string url = JsonConvert.DeserializeObject<string>(component.data.ToString());
+                            string assedId = url.Replace("ethereum://","");
```
Comment (sandrade-dcl) [inline unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/ManifestTranslator.cs]: Can we extract these kind of strings (`"ethereum://"`) into constants?

**Review by sandrade-dcl — APPROVED** [review]
It looks fine, good work!

Please do a commit with the pre-commit hook activated because I found several "format" cases that would be fine to fix. Thisngs like these:
image
image

File: unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/ManifestTranslator.cs (hunk 1 above)
Comment (Suduck) [reply unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/ManifestTranslator.cs:238]: Good catch!

File: unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/ManifestTranslator.cs (hunk 2 above)
Comment (Suduck) [reply unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/ManifestTranslator.cs]: Sure!

**Pull request merged**